playground/js-sourcemap/__tests__/js-sourcemap.spec.ts

```diff
@@ -36,6 +37,7 @@ if (!isBuild) {
     )
```

Comment:
Would you rename the test name to "js with inline sourcemap injected by a plugin"?

Reply:
Good name